### DISCUSSION TITLE
The geometry and the lattice size relation

### DISCUSSION THREAD

**Discussion by ztdepztdep** [CFD-GO/TCLB#371] [General]
If i set a 128*128 lattice size box for the lid-driven cavity. Does it mean the real geometry size of the cavity is 127*127.  and the border of the cavity is useless for the results visualization.

**Comment by llaniewski** [comment]
It depends on your setup. but if you have bounce back boundary conditions on both sides, then you'll have 126x126 real domain.

**Reply by ztdepztdep** [reply]
Does it mean the computation node is located in the cell center?

**Reply by llaniewski** [reply]
Yes.

Bounce Back boundary condition corresponds to a boundary half-way between two nodes. So if you have:
BCCCCCB
with B=Bounce Back, and C=Collision, you'll end up with 5 units distance between walls.
